src/Common/src/System/Net/WebSockets/ManagedWebSocket.cs

```diff
@@ -1279,6 +1283,14 @@ private void ThrowIfOperationInProgress(Task operationTask, [CallerMemberName] s
                 throw new InvalidOperationException(SR.Format(SR.net_Websockets_AlreadyOneOutstandingOperation, methodName));
             }
         }
+        
+        private void ThrowIfDisposed()
+        {
+            if (_disposed)
+            {
+                throw new ObjectDisposedException(GetType().FullName);
```

Comment:
We should decide on ONE pattern to use:

* GetType().ToString()
* GetType().FullName

cc: @safern @morganbr @tarekgh Any thoughts on which pattern is better?  i.e. Will both/either surive .NET Native process where strings are removed from assemblies?